app/assets/javascripts/miq_ujs_bindings.js

```diff
@@ -128,7 +128,7 @@ $(document).ready(function () {
     }
     miqJqueryRequest(url, options);
 
-    return false;
+    event.stopPropagation();
```

Comment:
Uh, can you also add `event` as an argument to the anonymous function you're in? (as it is, you're using global event instead of the one provided by jQuery to the event handler -  it would probably break only in ie9 but still.. ;-))